package.xml

```diff
@@ -1,17 +1,18 @@
 <?xml version="1.0"?>
-<package format="2">
+<package format="3">
   <name>vision_msgs</name>
   <version>1.0.0</version>
   <description>
     Messages for interfacing with various computer vision pipelines, such as
     object detectors.
   </description>
 
-  <author email="adam.d.allevato@gmail.com">Adam Allevato</author>
   <maintainer email="adam.d.allevato@gmail.com">Adam Allevato</maintainer>
 
   <license>Apache License 2.0</license>
 
+  <author email="adam.d.allevato@gmail.com">Adam Allevato</author>
```

Comment:
Just curious, is this change for `author` line required for format3?

Reply:
no sorry that was my OCD, this was invalid in format 2 as well:

```
$ xmllint package.xml -schema http://download.ros.org/schema/package_format2.xsd > /dev/null
package.xml:10: element author: Schemas validity error : Element 'author': This element is not expected. Expected is ( maintainer ).
package.xml fails to validate
```

The package would have compiled without the change